--- ProfitLib/ProfitLib.py~	2015-07-02 09:59:28.000000000 -0700
+++ ProfitLib/ProfitLib.py	2015-07-02 10:01:24.871067847 -0700
@@ -27,17 +27,17 @@
 import jsonrpc
 import sys
 from decimal import *
-sys.path.insert(0, './PyCryptsy/')
+sys.path.insert(0, './ProfitLib/PyCryptsy/')
 from PyCryptsy import PyCryptsy
-sys.path.insert(0, './python-bittrex/bittrex/')
+sys.path.insert(0, './ProfitLib/python-bittrex/bittrex/')
 from bittrex import Bittrex
-sys.path.insert(0, './PyCCEX/')
+sys.path.insert(0, './ProfitLib/PyCCEX/')
 from PyCCEX import PyCCEX
-sys.path.insert(0, './PyCryptopia/')
+sys.path.insert(0, './ProfitLib/PyCryptopia/')
 from PyCryptopia import PyCryptopia
-sys.path.insert(0, "./poloniex/")
+sys.path.insert(0, "./ProfitLib/poloniex/")
 from poloniex import poloniex
-sys.path.insert(0, "./bleuBot/")
+sys.path.insert(0, "./ProfitLib/bleuBot/")
 from bleuBot import bleuBot
 
 class ProfitLib:
